website/integrations/services/jellyfin/index.md

```diff
@@ -87,6 +89,89 @@ At this point, enter a username and click "Save Search Attribute Settings and Qu
 1. Click "Save"
 2. Logout, and login with a LDAP user. Username **must** be used, logging in with email will not work.
 
-## authentik Configuration
+### authentik Configuration
 
 No additional authentik configuration needs to be configured. Follow the LDAP outpost instructions to create an LDAP outpost and configure access via the outpost
+
+## OIDC Configuration
+### Jellyfin Configuration
+
+1. Navigate to your Jellyfin installation and log in with the admin account or currently configured local admin.
+2. Open the **Administrator dashboard** and go to the **Plugins** section.
+3. Then click the **Repositories** section at the top and add the below repository with the name of SSO-Auth
+
+```
+https://raw.githubusercontent.com/9p4/jellyfin-plugin-sso/manifest-release/manifest.json
+```
+
+4. Click the **Catalog** tab on top and install the SSO-Auth with the most recent version.
+5. Restart the Jellyfin server.
+6. Go back to the plugin tab.
+7. Click the SSO-Auth plugin.
+8. Fill out the Add / Update Provider Configuration:
+
+    - Name of OID Provider: `authentik`
+    - OID Endpoint: `https://authentik.company/application/o/jellyfin/.well-known/openid-configuration`
+    - OpenID Client ID: ClientID from provider
+    - OID Secret: Client Secret from provider
+    - Enabled: **CHECKED**
+    - Enable Authorization by Plugin: **CHECKED**
+
+9. If you want to use the role claim then also fill out these:
+
+    - Roles: roles to look for when authorizing access (should be done through authentik instead)
+    - Admin Roles: roles to look for when giving admin privilege
+    - Role Claim: `groups`
+
+10. Hit **Save** at the bottom.
+11. On the left side now click the **General** under dashboard and go to **Branding**.
+12. In the login disclaimer put this code and making sure to change the url at the top:
+
+```
+<form action="https://jellyfin.company/sso/OID/start/authentik">
+  <button class="raised block emby-button button-submit">
+    Sign in with SSO
+  </button>
+</form>
+```
+
+13. In the Custom CSS code also add this:
+
+```
+a.raised.emby-button {
+    padding:0.9em 1em;
+    color: inherit !important;
+}
+
+.disclaimerContainer{
+    display: block;
+}
+```
+
+14. Click **Save** at the bottom & restart the server.
+15. When you are signed out you should now see a **Sign in with SSO** button.
+
+:::note
+If you have problems check your logs which are under the **Administration** -> **Dashboard** then "logs" and will be near the bottom (most likely) with `Jellyfin.Plugin.SSO_Auth.` as the start of the lines you are looking for.
+:::
+
+### authentik Configuration
```

Comment:
It could be a good idea to start with the authentik config section first, so that client id and secret are available to be used during the jellyfin config directly.